README.md

```diff
@@ -170,6 +170,38 @@ jobs:
 
 </details>
 
+### Using pre-commit
+[pre-commit](https://pre-commit.com/) lets you easily install and manage pre-commit hooks.
+
+First, install pre-commit. Then, add the following to your `.pre-commit-config.yaml`
+
+```yaml
+-   repo: <link-to-repo>
+    rev: d4327e1f
```

Comment:
Should this point to a tag instead?

Reply:
Yes, that would be great. Please suggest a tag (maybe a new version tag) that will be assigned when the feature is merged, then we can add it here.

Reply:
I have updated the `rev` field to a new tag `v4.0.1`, you would have to add this (or change it and add a different one) after the changes are merged.